content/blog/how-to-provision-hpe-greenlake-private-cloud-resources-from-servicenow-service-catalog.md

```diff
@@ -0,0 +1,175 @@
+---
+title: How to provision HPE GreenLake private cloud resources from ServiceNow
+  Service Catalog
+date: 2022-03-08T06:13:00.000Z
+author: Vinnarasu Ganesan, Thavamaniraja S
+authorimage: /img/Avatar1.svg
+thumbnailimage: /img/thumbnail.jpg
+tags:
+  - hpe-greenlake
+---
+## Introduction
+
+A key strength of HPE GreenLake for private cloud is self-service
+orchestration and automation, which you do from the HPE GreenLake for
+private cloud dashboard and can be consumed using API and IaC. However,
+some ServiceNow users prefer to provision resources from the ServiceNow
+Service Catalog. For these users, HPE GreenLake for private cloud offers
+a free Morpheus plugin for ServiceNow, which they can install from the
+ServiceNow Store. After you install and configure the plugin, HPE
+GreenLake for private cloud catalog items (In the previous
+[blog](https://developer.hpe.com/blog/curate-and-expose-service-catalog-items-using-hpe-greenlake-for-private-cloud/),
+we discussed how to curate and expose service catalog items using
+HPE GreenLake for private cloud) can be presented in the ServiceNow
+Service Catalog for ordering. You turn on this integration by installing
+the Morpheus plugin on ServiceNow and sharing the wanted resources
+from HPE GreenLake for private cloud. This article walks through the
+process of integrating ServiceNow with HPE GreenLake for private cloud
+and exposing Service Catalog Items to ServiceNow.
+
+## Prerequisites
+
+The process of integrating HPE GreenLake for private cloud on ServiceNow
+needs the following:
+
+* Morpheus plugin on ServiceNow: To obtain and install the Morpheus
+  plugin, you must have your HI credentials.
+
```

Comment:
remove line break here after the HI credentials. and the "Using HI account...".

Reply:
A link on how to get these servicenow HI  credentials could be useful there.

Reply:
Getting ServiceNow HI credential is not in GLC. We expect the ServiceNow Administrator will have the credentials.